doc/Development_Documentation/04_Assets/07_Restricting_Public_Asset_Access.md

```diff
@@ -125,15 +125,16 @@ namespace App\Controller;
 
 use Pimcore\Controller\FrontendController;
 use Pimcore\Model\Asset;
+use Pimcore\Tool\Storage;
 use Symfony\Component\HttpFoundation\BinaryFileResponse;
 use Symfony\Component\HttpFoundation\Request;
 use Symfony\Component\HttpKernel\Exception\AccessDeniedHttpException;
 use Symfony\Component\Routing\Matcher\UrlMatcher;
-use Symfony\Component\Routing\RouteCollection;use Symfony\Component\Templating\Storage\Storage;
+use Symfony\Component\Routing\RouteCollection;
 
 class MyAssetController extends FrontendController
 {
-    public function protectedAssetAction(Request $request)
+    public function protectedAsset(Request $request)
```

Comment:
```suggestion
    public function protectedAssetAction(Request $request)
```

Reply:
@dvesh3 Ok, so I'm also going to update the docs in line 116, I guess?

Reply:
yes, please.